cpp/src/bindings.cpp

```diff
@@ -492,13 +501,19 @@ PYBIND11_MODULE(_pypowsybl, m) {
             });
 
     m.def("get_branch_flows_sensitivity_matrix", &pypowsybl::getBranchFlowsSensitivityMatrix, "Get sensitivity analysis result matrix for a given contingency",
-          py::arg("sensitivity_analysis_result_context"), py::arg("contingency_id"));
+              py::arg("sensitivity_analysis_result_context"), py::arg("matrix_id"), py::arg("contingency_id"));
+
+    m.def("get_precontingency_branch_flows_sensitivity_matrix", &pypowsybl::getPreContingencyBranchFlowsSensitivityMatrix, "Get sensitivity analysis result matrix for a given contingency",
```

Comment:
Why do we need 3 different methods get_branch_flows_sensitivity_matrix, get_precontingency_branch_flows_sensitivity_matrix, get_postcontingency_branch_flows_sensitivity_matrix ?
Matrix are uniquely identified by its ID, so no need to specify the contingency context.